### PR TITLE
Time and Number-Range detection test cases  in yaml format.

### PR DESCRIPTION
**Pull request by ashutoshsingh0223**

### CODE REVIEW

File: ner_v2/tests/numeral/number_range/test_number_range_detection.py [hunk 1]
```diff
@@ -1,63 +1,79 @@
...
+                                          language=language,)
+                    )
+
+    @classmethod
+    def get_yaml_test(cls, testcase, language, **kwargs):
+        def parse_expected_outputs(expected_outputs):
+            time_dicts, original_texts = [], []
+            for expected_output in expected_outputs:
+                time_dict = {
+                        "min_value": expected_output["min_value"],
```
Comment (srajdev) [inline ner_v2/tests/numeral/number_range/test_number_range_detection.py]: E126 continuation line over-indented for hanging indent

File: ner_v2/tests/numeral/number_range/test_number_range_detection.py [hunk 2]
```diff
@@ -1,63 +1,79 @@
...
+                        "unit": expected_output["unit"],
+                        "max_value": expected_output["max_value"],
+                }
+                original_text = expected_output["original_text"].lower().strip() if expected_output["original_text"] else None
```
Comment (srajdev) [inline ner_v2/tests/numeral/number_range/test_number_range_detection.py]: E501 line too long (126 > 119 characters)

File: ner_v2/tests/temporal/time/test_time_detection.py [hunk 3]
```diff
@@ -1,85 +1,93 @@
...
+                                          timezone=timezone)
+                    )
+
+    @classmethod
+    def get_yaml_test(cls, testcase, language, timezone, **kwargs):
+        def parse_expected_outputs(expected_outputs):
+            time_dicts, original_texts = [], []
+            for expected_output in expected_outputs:
+                time_dict = {
+                        "hh": expected_output["hh"],
```
Comment (srajdev) [inline ner_v2/tests/temporal/time/test_time_detection.py]: E126 continuation line over-indented for hanging indent

File: ner_v2/tests/numeral/number_range/test_number_range_detection.py [hunk 4]
```diff
@@ -1,63 +1,80 @@
 from __future__ import absolute_import
-
-from django.test import TestCase
-import pandas as pd
+import io
 import os
+import six
+import yaml
+from django.test import TestCase
```
Comment (srajdev) [inline ner_v2/tests/numeral/number_range/test_number_range_detection.py]: F401 'django.test.TestCase' imported but unused

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Pull request closed without merging**